guides/v2.1/frontend-dev-guide/css-guide/css_quick_guide_mode.md

```diff
@@ -44,7 +44,7 @@ The following is an illustration of how the process of making simple changes loo
 
     ![Less code redefining the color of the primary buttons]({{ site.baseurl }}/common/images/extend_less_screenshot121.png)
 
-1. Change the button font by adding the following code in the `_extend.less` file:
+1. Change the button font size by adding the following code in the `_extend.less` file:
```

Comment:
Please add these changes to v2.2 as well, as v2.3 is symlinked. This way, all versions will get the change.

Reply:
Your PR, #2179, should be merged with this one.